catalog/src/main/kotlin/com/adevinta/spark/catalog/CatalogApp.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023 Adevinta
+ * Copyright (c) 2023-2024 Adevinta
```

Comment:
@soulcramer can you check your IDE if you don't have any custom copyright code, because this is not the expected format 🙈

Reply:
Damn why is this coming back 💀
I'll check